### ISSUE TITLE
Rebuild for the python3.6

### ISSUE THREAD

**Issue by Meloneat** [dzone/osqa#521]
l begin to try the Django program and l use the python 3.6, but l get too many mistakes about the reference and like  'cannot import name xx' when l debug the program, so could we pull a new branch to  request optimizing the osqa for the new version python.After all ,the osqa source had been last commited three years ago.And the Django have been updated the version 2.0,why not give new life to osqa?

**Comment by davidljung** [comment]
Agreed.  It doesn't reflect well on Answerhub either.   We're a large company and are evaluating OSQA.  We'd thought to perhaps deploy OSQA on AWS initially, then see how it goes and consider switching over to Answerhub to gain features, but given the codebase here doesn't even run on Python 3.5.2 and isn't straightforward to install, it doesn't give much faith in the answerhub code base being actively maintained and developed.
